src/library/playlistfeature.cpp

```diff
@@ -189,15 +189,31 @@ void PlaylistFeature::slotPlaylistTableChanged(int playlistId) {
     enum PlaylistDAO::HiddenType type = m_playlistDao.getHiddenType(playlistId);
     if (type == PlaylistDAO::PLHT_NOT_HIDDEN ||
         type == PlaylistDAO::PLHT_UNKNOWN) { // In case of a deleted Playlist
+        // TODO: Is this even necessary?
         clearChildModel();
         m_lastRightClickedIndex = constructChildModel(playlistId);
+    }
+}
+
+void PlaylistFeature::slotPlaylistTableRenamed(int playlistId,
```

Comment:
but there has to be a conditional to see if the playlist is in the current object's list.  That was the issue I had before.  PlaylistFeature and SetLogFeature use different checks